_inc/client/components/module-settings/index.jsx

```diff
@@ -502,6 +502,23 @@ export let VerificationToolsSettings = React.createClass( {
 						</span>
 					</div>
 
+					<div className="dops-card">
+						<FormLabel>
+							<FormLegend>Yandex</FormLegend>
+							<TextInput
+								name={ 'yandex' }
+								value={ this.props.getOptionValue( 'yandex' ) }
+								placeholder={ 'Example: 44d68e1216009f40' }
+								className="widefat code"
+								disabled={ this.props.isUpdating( 'yandex' ) }
+								onChange={ this.props.onOptionChange} />
+						</FormLabel>
+						<span className="jp-form-setting-explanation">
+							{ __( 'Meta key example: ' ) }
```

Comment:
Agreed. Removed space here and in few other places in this file. 462db814f9660e49bb00eff953fbc1a4c07a3945